### PR TITLE
Update contributor

### PR DESCRIPTION
**Pull request by panford**
This PR adds InstaDeep's AI for social good as a contributor to the SKAI project.
Image from this [link](https://drive.google.com/file/d/1UaizEHM4V-PSSKphoIFvTnSa7e1dOt1Z/view?usp=sharing) could be added to the skai-public storage and link on last line of the README file updated.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -29,4 +29,9 @@ Please see detailed instructions [here](assessment_instructions.md).
...
 
-This software was developed in collaboration with the United Nations World Food Programme (WFP) Innovation Accelerator. The WFP Innovation Accelerator identifies, supports and scales high potential solutions to hunger worldwide. The Innovation Accelerator supports WFP innovators and external start-ups and companies through financial support, access to a network of experts and a global field reach.
+This software was developed in collaboration with the United Nations World Food Programme (WFP) Innovation Accelerator and [InstaDeep](https://www.instadeep.com/)'s AI for Social Good (AI4SG) team.
+
+The WFP Innovation Accelerator identifies, supports and scales high potential solutions to hunger worldwide. The Innovation Accelerator supports WFP innovators and external start-ups and companies through financial support, access to a network of experts and a global field reach.
+InstaDeep's AI for Social Good (AI4SG) team utilizes the company's expertise in AI research and engineering to create technologies aimed at improving human welfare and enhancing global well-being.
+
+![collaborators logos](https://drive.google.com/file/d/1UaizEHM4V-PSSKphoIFvTnSa7e1dOt1Z/view?usp=sharing)
```
Comment (jzxu) [inline docs/README.md:37]: Can you add this image file to the repo? Referencing a Google drive link seems a bit unstable.

File: docs/README.md [hunk 2]
```diff
@@ -29,4 +29,9 @@ Please see detailed instructions [here](assessment_instructions.md).
 
 ## Acknowledgments
 
-This software was developed in collaboration with the United Nations World Food Programme (WFP) Innovation Accelerator. The WFP Innovation Accelerator identifies, supports and scales high potential solutions to hunger worldwide. The Innovation Accelerator supports WFP innovators and external start-ups and companies through financial support, access to a network of experts and a global field reach.
+This software was developed in collaboration with the United Nations World Food Programme (WFP) Innovation Accelerator and [InstaDeep](https://www.instadeep.com/)'s AI for Social Good (AI4SG) team.
```
Comment (jzxu) [inline docs/README.md:32]: What do you think about structuring it like this?

```
This software was developed in collaboration with the following organizations:

### United Nations World Food Program (WFP) Innovation Accelerator

The WFP Innovation Accelerator identifies, supports and scales high potential solutions to hunger worldwide. The Innovation Accelerator supports WFP innovators and external start-ups and companies through financial support, access to a network of experts and a global field reach.

### InstaDeep AI for Social Good (AI4SG) team

InstaDeep's AI for Social Good (AI4SG) team utilizes the company's expertise in AI research and engineering to create technologies aimed at improving human welfare and enhancing global well-being.
```

**Review by jzxu — CHANGES_REQUESTED** [review]
Thanks for this PR!

File: docs/README.md (hunk 1 above)
Comment (panford) [reply docs/README.md:37]: Hi @jzxu 
I added that link as a placeholder. I see there's a storage (https://storage.googleapis.com/skai-public/documentation_images) for other images used in the docs. Wondering if you could add this as well as I don't have access.

File: docs/README.md (hunk 2 above)
Comment (panford) [reply docs/README.md:32]: I like this!
Doing the update, thanks

**Review by panford — COMMENTED** [review]
Thank you @jzxu 
The structure suggested has been applied.

File: docs/README.md (hunk 1 above)
Comment (panford) [reply docs/README.md]: Thank you @jzxu 
I have added the image to the repo as suggested.

**Pull request merged**